dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -312,11 +314,156 @@ def build_ci_image(
 
 @option_verbose
 @main.command(name='build-prod-image')
-def build_prod_image(verbose: bool):
+@click.option(
```

Comment:
I think it woudl be great to add ENVVARS for all the parameters here already similarly to what @edithturn added in CI build.